pkg/capacity/options.go

```diff
@@ -0,0 +1,18 @@
+package capacity
+
+type Option struct {
```

Comment:
`
$ golangci-lint run

pkg/capacity/options.go:3:6: exported type `Option` should have comment or be unexported (golint)
type Option struct {
     ^
`
there is some lint here

Reply:
Comments have been added